src/components/map/UserCard.tsx

```diff
@@ -21,6 +21,8 @@ const UserCard = ({ user, onClose }: UserCardProps) => {
   const [lastActive, setLastActive] = useState(user.lastActive); // TODO: setLastActive when user is active?
   const [location, setLocation] = useState(user.lastActive.location);
   const [relativeTimeString, setRelativeTimeString] = useState('...');
+
+  // TODO: Update status color based on lastActive
   const [statusColor, setStatusColor] = useState(COLORS.NIGHTLIGHT_GRAY);
```

Comment:
there should be a util for this as welll

Reply:
no backend for this yet. TODO for future